app/src/main/res/layout/fragment_filter_menu.xml

```diff
@@ -56,20 +56,35 @@
         android:layout_height="40dp"
         android:layout_marginTop="188dp"
         android:background="@color/zxing_transparent"
+        android:src="@drawable/filter_menu"
```

Comment:
is this being used, did you intend to add src to this button?

Reply:
Whoops, left in by mistake. Nice catch